jib-core/src/test/java/com/google/cloud/tools/jib/cache/LayerEntriesSelectorTest.java

```diff
@@ -63,6 +67,29 @@
         .collect(ImmutableList.toImmutableList());
   }
 
+  @Test
+  public void testLayerEntryTemplate_fileModified() throws IOException {
+    TemporaryFolder temporaryFolder = new TemporaryFolder();
```

Comment:
I would recommend using `@Rule` since it would run the cleanup of the created folder automatically.